components/api/core/utils.bal

```diff
@@ -39,3 +40,81 @@ function updatePayloadWithUserInfo (json payload, string field) returns error? {
         log:printDebug(io:sprintf("Response modification failed : User information not found for user : \'%s\'", userId));
     }
 }
+
+function getUserToken(string authorizationHeader, string cookieHeader) returns string {
+    string token = "";
+    string[] splittedToken = authorizationHeader.split(" ");
+    if splittedToken.length() != 2 || !(splittedToken[0].equalsIgnoreCase(constants:BEARER_HEADER)) {
+        log:printError("Did not receive the token in proper format");
+    }
+    string lastTokenElement = splittedToken[1];
+    string | error firstTokenElement = filter:getCookie(cookieHeader);
+    if (firstTokenElement is error) {
+        log:printError("Cookie value could not be resolved. Passing to the next filter", err = firstTokenElement);
+    } else {
+        token = io:sprintf("%s%s", firstTokenElement, lastTokenElement);
+        if "" == token {
+            log:printDebug("Did not receive any token. Passing the request to the next filter");
```

Comment:
This debug log should not be printed here. Since this is now a util method, it cannot mention a filter.